cookbooks/bcpc-hadoop/attributes/default.rb

```diff
@@ -49,6 +49,30 @@
 default["bcpc"]["hadoop"]["hbase"]["superusers"] = ["hbase"]
 # Interval in milli seconds when HBase major compaction need to be run. Disabled by default
 default["bcpc"]["hadoop"]["hbase"]["major_compact"]["time"] = 0
+default["bcpc"]["hadoop"]["hbase"]["bucketcache"]["enabled"] = false
```

Comment:
Thanks @kiiranh . Looking at the latest code, we seem to be breaking the attributes on component lines. It is a good to break at this stage irrespective of number if attributes.